### PR TITLE
Improved offline installation instructions to point to correct installation scripts and clarify process

### PR DESCRIPTION
**Pull request by IceCodeNew**
##### Summary
Fix a bug in the document. (https://github.com/netdata/netdata/blame/9342704a41cbd20a315e55f0b1ff3efd9f039498/packaging/installer/methods/offline.md#L37)
Also make the part that should be replaced accordingly in the example command clearer. (https://github.com/IceCodeNew/netdata/commit/e13d9a60a6d587ae80b2bfc7326b9a809aa48d69#diff-40714f9627176dfafc102507b4ffb471R73)

##### Component Name
None.

##### Test Plan
None. It's just a rewording of the document.

##### Additional Information
None.

### CODE REVIEW

File: packaging/installer/methods/offline.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ curl -s https://api.github.com/repos/netdata/netdata/releases/latest | grep "bro
 curl -s https://api.github.com/repos/netdata/netdata/releases/latest | grep "browser_download_url.*txt" | cut -d '"' -f 4 | wget -qi -
 
 # Netdata dependency handling script
-curl -s https://raw.githubusercontent.com/netdata/netdata-demo-site/master/install-required-packages.sh | wget -qi -
+wget -q - https://raw.githubusercontent.com/netdata/netdata-demo-site/master/install-required-packages.sh
```
Comment (prologic) [inline packaging/installer/methods/offline.md:37]: The correct link to this script is actually:

https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh

Can you amend this?

Comment (IceCodeNew) [reply packaging/installer/methods/offline.md:37]: Sure

**Comment by ilyam8** [comment]
@IceCodeNew could you rebase on current master? to get netlify errors fixed

**Comment by IceCodeNew** [comment]
> @IceCodeNew could you rebase on current master? to get netlify errors fixed

You got it.

**Comment by joelhans** [comment]
@IceCodeNew I'm not sure why, but the rebase didn't seem to fix the Netlify error.

We updated our [requirements.txt](https://github.com/netdata/netdata/blob/master/docs/generator/requirements.txt) file that's at `docs/generator/requirements.txt` with the following:

```
mkdocs>=1.0.1
mkdocs-material==4.6.3
```

You could try rebasing again, or edit that same file on your branch to align them. Sorry for the inconvenience.

I also retitled your PR so that when it's merged it has some more at-a-glance value to anyone who might be reading the changelog.

**Comment by ilyam8** [comment]
`mkdocs-material` is not updated in the @IceCodeNew fork
https://github.com/IceCodeNew/netdata/blob/master/docs/generator/requirements.txt

**Comment by IceCodeNew** [comment]
> You could try rebasing again, or edit that same file on your branch to align them. Sorry for the inconvenience.

I see. Will align it soon.

> I also retitled your PR so that when it's merged it has some more at-a-glance value to anyone who might be reading the changelog.

Thanks. I learned so much about how to contribute efficiently and how exactly high-quality PR should be.

**Comment by IceCodeNew** [comment]
Not finish yet. I will delete this comment while I've been ready for new PR.

**Comment by IceCodeNew** [comment]
```
git fetch upstream
git checkout master
git rebase upstream/master
git push -f
```

I used to issue `git merge upstream/master` to sync repo, but this time I was told to use rebase. Found that the commit history LGTM, so I think that would be my new PR.

**Pull request closed without merging**